src/legacy/core_plugins/kibana/index.js

```diff
@@ -157,6 +157,7 @@ export default function (kibana) {
           kbnBaseUrl,
           uiCapabilities: {
             discover: {
+              show: true,
```

Comment:
Yeah, it's redundant for sure; however, I feel that it's inline with the privileges being tied to multiple different "actions" so that the consumer of the actions can simply check what they're concerned with without knowing about other parts of the system. It allows us some flexibility, but in this situation the likelihood of us wanting to view an app when the navlink is hidden is super low, but there is potential for a sub-app (like advanced settings in management) to not have a navLink so this separation helps there.